docs/_content/ontwikkelaars/handleidingen-en-tutorials/large-files.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Large files upload tutorial"
+weight: 100
+---
+
+In this tutorial we explore the ways to upload files to DRC depending on the file size. 
+
+The most relevant component for this tutorial:
+
+* DRC: for creating documents and uploading files
+
+## The requirements for this tutorial
+
+* `docker` and `docker-compose` to host the components locally on your (development machine). 
+  See [installatie en configuratie](./installatie-en-configuratie) for a detailed description.
+* The [eenmalige setup](./eenmalige-setup) has been carried out.
+
+## Overview of file upload process in DRC
+
+The main resource of DRC API is `EnkelvoudigInformatieObject` which supports creating and editing documents. 
+It's content can be divided into metadata (`identificatie`, `creatiedatum`, `titel` and etc) and file content.
+The main metadata of file content used by DRC is file size derived from `bestandsomvang` attribute. 
+The process of uploading files is chosen based on the comparision of the value of `bestandsomvang` and the maximum file size.
+
+There can be 3 options:
+
+* file size = 0, i.e. document contains only metadata without file content. Document is created during 1 request to DRC.
```

Comment:
I would probably stick to `EnkelvoudigInformatieObject` (or InformatieObject) instead of using Document, as using both interchangeably might be confusing to some